changelog.md

```diff
@@ -1,5 +1,111 @@
 ## 15.x
 
+### 15.0-beta2 (2025-04-13)
+
+- Feature: Snow-covered rocks are now visible (#13627)
+- Feature: Generate more rocks on steep slopes during map generation or heightmap import (#13462)
+- Feature: Prevent towns from upgrading individually-placed houses (#13270)
+- Feature: [Win32] Touchpad two-finger map scrolling (#13172)
+- Feature: NewGRF Badges (#13073)
+- Add: [NewGRF] Add road-/tram-/rail-type variable 0x45 to get mutual road-/tram-/rail-type on same tile (#13934)
+- Add: [Script] Newer Cargo Classes (#13779)
+- Add: Show hyperlink destination tooltips in text window (#13742)
+- Add: [Script] Saving/loading ScriptList (#13556)
+- Add: Press Ctrl to build diagonal canals in game mode (#13432)
+- Add: Sandbox settings to Sandbox Options window (#13268)
+- Add: Setting to allow placing houses manually in-game (#13266)
+- Add: [Script] Event for when a company's president name changes (#13208)
+- Add: Ability to toggle visibility of station signs by facility (#13207)
+- Add: [Script] ScriptEventCompanyRename (#12878)
+- Add: Ukrainian Hryvnia currency (#12877)
+- Add: Convert 32bpp-only sprites to 8bpp when needed (#11602)
+- Change: [Script] Start GS (but don't run it) when generating world in scenario editor (#13961)
+- Change: Make tree placement at world generation look more organic (#13515)
+- Change: [MacOS] Put the icon in a rounded rectangle (#13446)
+- Change: [Script] GetWaypointID to return the StationID of any waypoint (#13407)
+- Change: Draw company manager face jacket after collar (#13390)
+- Change: Don't distinguish between bus and truck stops when removing them (#13384)
+- Change: [Script] Rename BridgeID to BridgeType in the script API (#13352)
+- Change: Add fonts document to help window (#13305)
+- Change: Log changes to sandbox settings (#13267)
+- Change: When player joins network company, use its name instead of number in chat (#13263)
+- Change: [Win32] Draw window title bar according to current Windows light/dark theme (#13196)
+- Change: Restore wider spacers in main toolbars (#12039)
+- Fix: NewGRF Global variables 0D, 0E and 1E refer to wrong GRFFile (#13986)
+- Fix #13980: Allow diagonal selection for road convert (#13983)
+- Fix: Validate raw strings from game-scripts, and strip invalid and control characters (#13976)
+- Fix: Capitalise "Disabled" for the "maximum non-sticky open windows" setting (#13975)
+- Fix: Fixed accidental changes to autorail/road AI behavior (#13968)
+- Fix: Frame widget with label had incorrect spacing (#13967)
+- Fix: StringFilter included quotes in the search and failed (#13965)
```

Comment:
This PR number talks about something else: Replace Utf8Encode and other C-style string construction with bounds checking StringBuilder class

Reply:
It's one of the commits from that PR, should have split better.

Reply:
The commit is a part of that PR. Not sure if this is actually just a fix inside the PR, so I could drop it.